### PR TITLE
hello-world-api: add basic volume test

### PR DESCRIPTION
**Pull request by marijanp**
This PR introduces a basic volume-test for `hello-world-api`. It answers the question whether our contracts are executed for a certain amount of wallets.
This number is currently 10 because there is an issue with CTL when making a lot of requests see: https://github.com/Plutonomicon/cardano-transaction-lib/issues/1063

A check output for the volume test was created. The test-executable decides whether it should execute the volume tests or something else depending on a new environment variable

I also added/ modified some of the log-messages, which I found useful when debugging the volume-test

other changes

- [ ] The impure tests via `nix run .#offchain:test` were run and all are passing.
- [ ] Where applicable, grammar, punctuation, and spelling within code and comments should be plausibly correct.
- [ ] The issue(s) that the work is addressing should be linked in the PR description.
- [ ] Any significant changes beyond what's clear from the linked issue(s) are documented in the PR description

### CODE REVIEW

File: offchain/hello-world-api/src/Api.purs [hunk 1]
```diff
@@ -41,23 +41,31 @@ import Util (buildBalanceSignAndSubmitTx, waitForTx, getUtxos, decodeCbor, getDa
...
+  logDebug' $ "incremented hello-world: param: " <> show param <> " newDatum: " <> show newDatum
+  pure txIn
 
 redeem :: Int -> TransactionInput -> Contract () Unit
 redeem param lastOutput = do
+  logDebug' $ "redeeming hello-world: param: " <> show param
   validator <- helloScript param
   let vhash = validatorHash validator
   redeemFromScript vhash validator lastOutput
+  logDebug' $ "redeemed hello-world: param: " <> show param
```
Comment (epiro-ps) [inline offchain/hello-world-api/src/Api.purs:68]: Are these logs left here intentionally?

**Review by epiro-ps — CHANGES_REQUESTED** [review]
Looks good, just a potentially non-blocking question on the logs here.

File: offchain/hello-world-api/src/Api.purs (hunk 1 above)
Comment (marijanp) [reply offchain/hello-world-api/src/Api.purs:68]: Yes, they were helpful while debugging

**Review by epiro-ps — APPROVED** [review]
:fire: :fire: :fire:

**Review by Geometer1729 — APPROVED** [review]
LGTM
Did the plutip exit issue get sorted out?
Edit: never mind just spoted that in the description

**Pull request merged**